### ISSUE TITLE
Normal User - Seeing information about routers they do not have permission to see anything about under device -> ports 

### ISSUE THREAD

**Issue by eikik87** [kahun/awesome-sysadmin#368]
I want to give my customers access to use Librenms. But I only want to limit the user to see specific routers.

When I do that, and log in as that user. Under Devices/Ports on the device, I can see information on the right side about routers within the same Subnet or using same RFC 1918 subnet. Then it lists up our MGMT IP addresses/hostnames that we use in Librenms. 

So for example Vlan 1, that is on many routers. And they all have 192.168.1.0/24 subnet. So that router sees every single other router information because they are using same IP subnet.

That is ok, except I only want the routers he has access to too be listed up there. Not those he does not have access too.

There is one other router in this picture that is 'bold' that he has access too. And it is active link. Other routers are just gray and not link.

![image](https://cloud.githubusercontent.com/assets/13901674/13180260/9f9d545e-d71f-11e5-9769-f5c54ba5fad5.png)

**Comment by n1trux** [comment]
This is not a support ticket system for Librenms.

Try one of their support channels: http://www.librenms.org/#support

**Comment by eikik87** [comment]
I don't see how this is a 'support ticket' issue. This is clearly a problem with the code. Why would you let a user see information about equipment on the server that I specifically do not allow him access too.

**Comment by n1trux** [comment]
Yes, but this is the awesome-sysadmin repository, which has nothing to do with LibreNMS. Check out the link I sent you and/or create an issue on their bug tracker: https://github.com/librenms/librenms/issues

**Comment by eikik87** [comment]
lol what the heck...

**Comment by followsound** [comment]
This repository is simply to link to other projects, it is not actually the project itself.

As @n1trux has said https://github.com/librenms/librenms/issues will be a better place to post this

**Comment by eikik87** [comment]
Ye sorry about that, I have no idea how this got on some other forum. Already posted other issues on librenms github. You can close this issue. Posted it on Librenms github now.
